### PR TITLE
Added Unreal Engine 5 support

### PR DESCRIPTION
**Pull request by jezzye13**

### CODE REVIEW

File: templates/ue5_build.Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,76 @@
+// Fetch Shared Library from Github
+library identifier: 'JenkinsSharedLib@master',
+    retriever: modernSCM([
+      $class: 'GitSCMSource',
+      credentialsId: '', // Public repo, no credentials needed
+      remote: 'https://github.com/jezzye13/JenkinsSharedLib'
```
Comment (DavidtKate) [inline templates/ue5_build.Jenkinsfile]: remote url should be "https://github.com/DavidtKate/JenkinsSharedLib" to fit appropriate repo which template is stored in

**Review by DavidtKate — CHANGES_REQUESTED** [review]
Thank you for your contribution, I only have one suggested change before I merge this pull request :).

File: templates/ue5_build.Jenkinsfile (hunk 1 above)
Comment (jezzye13) [reply templates/ue5_build.Jenkinsfile]: Ja dat is logisch ik wist niet zeker hoe snel jij zou reageren op mijn pull request.

**Comment by jezzye13** [comment]
I made the changes of the repo URL in the templates.

**Pull request merged**